nemo/utils/exp_manager.py

```diff
@@ -1122,6 +900,9 @@ def configure_checkpointing(
     if 'mp_rank' in checkpoint_callback.last_model_path or 'tp_rank' in checkpoint_callback.last_model_path:
         checkpoint_callback.last_model_path = uninject_model_parallel_rank(checkpoint_callback.last_model_path)
     trainer.callbacks.append(checkpoint_callback)
+    # TODO create support_preemption arg, if True do the below
+    preemption_callback = PreemptionCallback(torch.device('cuda'), checkpoint_callback)
```

Comment:
Is there any reason to not have this callback?

Reply:
Preemption callback is enabled by default, have mentioned it in the docstring of the class.